### ISSUE TITLE
Requires ADMINISTRATOR privelage.

### ISSUE THREAD

**Issue by crawshaws** [ateodorescu/home-assistant-addons#1]
The ipmitool server seems to require ADMINISTRATOR privelages. It would be nice to be able to specify the level (such as USER) as connections fail when the IPMI user is restricted to anything other than USER

**Comment by ateodorescu** [comment]
I'm not sure what can be fetched from the IPMI server when connecting to it as a USER instead of an ADMINISTRATOR.

**Comment by crawshaws** [comment]
I am able to read all of the sensors, I did find one problem when going through the source code though. If using pyipmi, they specifically require the user to be an administrator as they do not have it as a configurable option.

I have set my user to a read-only admin for now. But it would be a nice to have in the future.

I have amended the code for the addon when using ipmitool and the integration to allow you to specify the level, but as I say, it will not work if pyipmi is to be used.

I just need to fork your repo, and test the changes somehow.
